src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/commands.py

```diff
@@ -17,13 +17,13 @@
 def load_command_table(self, _):
 
     container_services_sdk = CliCommandType(
-        operations_tmpl='azure.mgmt.containerservice.operations.'
+        operations_tmpl='azure.mgmt.containerservice.v2017_07_01.operations.'
                         'container_services_operations#ContainerServicesOperations.{}',
         client_factory=cf_container_services
     )
 
     managed_clusters_sdk = CliCommandType(
-        operations_tmpl='azure.mgmt.containerservice.operations.'
+        operations_tmpl='azure.mgmt.containerservice.v2018_03_31.operations.'
```

Comment:
This is a red flag. Why are you putting specific API versions in here?

Reply:
since the operations I needed is under azure.mgmt.containerservice.v2018_03_31.operations.

Reply:
There is no module named azure.mgmt.containerservice.operations, all operations folders are under specific version folders.

Reply:
If ACR has become a multi-API SDK which is what this looks like (@lmazuel can you confirm?) then this is not the correct way to do imports. You will need to make changes to the core CLI and update the imports throughout the ACR module.

Reply:
What do you mean core CLI? azure.core.cli?

Reply:
We have a similar question in PR #8075 for multi-api support, so let's make sure we're on the same page as @julienstroheker. See https://github.com/Azure/azure-cli/pull/8075/files#diff-91f1f79fb5cebd957f4940b4a4d17635R22